scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -54,23 +54,13 @@ object Common {
   def unrollTaps(step: FlowStep[JobConf]): Seq[Tap[_, _, _]] =
     unrollTaps(step.getSources.asScala.toSeq)
 
-  /**
-   * Get the total size of the input paths, which may contain a glob
-   * pattern in its path, so we must be ready to handle that case.
-   */
-  def inputSizes(step: FlowStep[JobConf]): Seq[(Path, Long)] = {
+  def inputSizes(step: FlowStep[JobConf]): Seq[(String, Long)] = {
     val conf = step.getConfig
-
-    FileInputFormat
-      .getInputPaths(conf)
-      .map { path =>
-        val fs = path.getFileSystem(conf)
-        val size = fs.globStatus(path)
-          .map(status => fs.getContentSummary(status.getPath).getLength)
-          .sum
-
-        path -> size
-      }
+    unrollTaps(step) flatMap {
```

Comment:
seems like we currently only fail if:
1) There's some exception thrown while computing the size of a tap
2) We can't compute the size of all the taps in a step. In case case we end up with an empty list and log the warning. 

If I understand correctly, this has been the prior behavior. Do we want to log a warning / fail if we can't compute the size of 1 of these n taps? I guess the ones we're explicitly skipping now are just the non-Hfs ones so just logging should be ok?